src/pages/participantList.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import SingleParticipant from "../components/SingleParticipant"
+import tempParData from "../tempData/participants"
+
+
+function ParticipantList() {
+
+    const participantList = tempParData.map(item => <SingleParticipant key={item.pid} par={item}/>)
+
+    return (
+        <div className="General">
+            <h3>Name: | Duty Prefrences: | Age:</h3> 
+            {participantList}
```

Comment:
I was taught never to use tables unless I actually _need_ a table. This is one of those :)